content/en/continuous_integration/intelligent_test_runner/how_it_works.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How It Works
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Datadog Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+
+## What is Intelligent Test Runner?
```

Comment:
re: warning - it is using sentence-style capitalization. Intelligent Test Runner is the proper name of the product.